ivy/functional/frontends/torch/linalg.py

```diff
@@ -170,3 +170,7 @@ def eig(input, *, out=None):
 @with_unsupported_dtypes({"1.11.0 and below": ("bfloat16", "float16")}, "torch")
 def solve(input, other, *, out=None):
     return ivy.solve(input, other, out=out)
+ 
+@to_ivy_arrays_and_back
+def vander(A, *, increasing=False, out=None):
+    return ivy.vander(A, out=out)
```

Comment:
both kwargs (`N` and `increasing`) should be dealt with

Reply:
so it should be this ? 

def vander(x, *, increasing=False, N=None):
    return ivy.vander(x, increasing=increasing, out=N)

Reply:
You can check the docs here: https://pytorch.org/docs/1.11/generated/torch.vander.html?highlight=vander#torch.vander

the order of the kwargs should be the same, and you should deal with any arguments that are not matched in the `ivy` function in the frontend function (which I guess it's the case with `N`) :slightly_smiling_face: